src/pymovements/datasets/didec.py

```diff
@@ -122,6 +125,8 @@ class DIDEC(DatasetDefinition):
 
     name: str = 'DIDEC'
 
+    long_name: str = 'The Dutch Image Description and Eye-tracking Corpus'
```

Comment:
as above: please remove the leading `The `

Reply:
done